### ISSUE TITLE
E: systemd-service-without-service_del_postun triggered improperly

### ISSUE THREAD

**Issue by FilippoBonazziSUSE** [rpm-software-management/rpmlint#1188]
I have this rpmlint error in a package:
```
[   71s] greetd.x86_64: E: systemd-service-without-service_del_postun greetd.service
[   71s] The package contains a systemd service but doesn't contain a %postun with a
[   71s] call to service_del_postun.
```

However the spec file contains:
```
%postun
%service_del_postun_without_restart %{name}.service
```

So it looks to me like the error should not be triggered.

**Comment by danigm** [comment]
It's in the `systemd-rpm-macros` pacakge, so I think it's something valid. The [documentation says](https://en.opensuse.org/openSUSE:Systemd_packaging_guidelines#Registering_unit_files_in_install_scripts) that it's okay to use this macro:

> During package update, %service_del_postun restarts units. If units are not to be restarted, %service_del_postun_without_restart should be used instead. 

So as far as I know, we should allow it.

**Comment by danigm** [comment]
Fixed by https://github.com/rpm-software-management/rpmlint/pull/1189
